pom.xml

```diff
@@ -198,7 +198,7 @@
         <dependency>
             <groupId>uk.gov.pay</groupId>
             <artifactId>utils</artifactId>
-            <version>1.0.0-6</version>
+            <version>1.0.0-f6097986677849386f283bb84072198b57931b1b</version>
```

Comment:
this can be replaced with `${pay-java-commons.version}`

Reply:
So it can. Done.